operator_ui/src/pages/JobsIndex/JobsIndex.tsx

```diff
@@ -42,22 +42,39 @@ interface Job<T> {
 }
 
 export type DirectRequest = Job<models.JobSpec>
-export type OffChainReporting = Job<models.OcrJobSpec>
-export type CombinedJobs = DirectRequest | OffChainReporting
+export type JobSpecV2 = Job<models.JobSpecV2>
+export type CombinedJobs = DirectRequest | JobSpecV2
 
-function isDirectRequest(job: CombinedJobs): job is DirectRequest {
-  return job.type === JobSpecTypes.jobSpec
+function isJobSpecV1(job: CombinedJobs): job is DirectRequest {
+  return job.type === JobSpecTypes.v1
 }
 
-function isOffChainReporting(job: CombinedJobs): job is OffChainReporting {
-  return job.type === JobSpecTypes.ocrJobSpec
+function isJobSpecV2(job: CombinedJobs): job is JobSpecV2 {
+  return job.type === JobSpecTypes.v2
+}
+
+function isFluxMonitorJobSpecV2(job: JobSpecV2) {
+  return job.attributes.type === 'fluxmonitor'
+}
+
+function isOCRJobSpecV2(job: JobSpecV2) {
+  return job.attributes.type === 'offchainreporting'
 }
 
 function getCreatedAt(job: CombinedJobs) {
-  if (isDirectRequest(job)) {
+  if (isJobSpecV1(job)) {
     return job.attributes.createdAt
-  } else if (isOffChainReporting(job)) {
-    return job.attributes.offChainReportingOracleSpec.createdAt
+  } else if (isJobSpecV2(job)) {
+    switch (job.attributes.type) {
+      case 'offchainreporting':
+        return job.attributes.offChainReportingOracleSpec.createdAt
+
+      case 'fluxmonitor':
+        return job.attributes.fluxMonitorSpec.createdAt
+
+      case 'directrequest':
+        return job.attributes.directRequestSpec.createdAt
+    }
```

Comment:
Seems like https://github.com/smartcontractkit/chainlink/pull/3981/files#diff-d1ba52d002bf42ed9cc5b63f76ee3eab3352ed574832f6730db79b9da12c823aR14. If `createdAt` doesn't get brought up to the job level (which I think it should) this is gonna be a common op so might be worth creating v2 job utils.

I'd still prefer `createdAt` to live on `job`.